### PR TITLE
Bugfix/permissions to outfile

### PR DESCRIPTION
**Pull request by AlonZivony**
## Initial Checklist

- [x] There is an issue describing the need for this PR.
- [x] Git log contains summary of the change.
- [x] Git log contains motivation and context of the change.
- [ ] If part of an EPIC, PR git log contains EPIC number.
- [ ] If part of an EPIC, PR was added to EPIC description.

## Description (git log)

Fix some capabilities problems and change access to output dir to drop `CAP_DAC_OVERRIDE`

Fixes: #1981 

## Type of change

- [x] Bug fix (non-breaking change fixing an issue, preferable).
- [ ] Quick fix (minor non-breaking change requiring no issue, use with care)
- [ ] Code refactor (code improvement and/or code removal)
- [ ] New feature (non-breaking change adding functionality).
- [ ] Breaking change (cause existing functionality not to work as expected).

## Final Checklist:

Pick "Bug Fix" or "Feature", delete the other and mark appropriate checks.

- [x] I have made corresponding changes to the documentation.
- [x] My code follows the style guidelines (C and Go) of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented all functions/methods created explaining what they do.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix, or feature, is effective.
- [x] New and existing unit tests pass locally with my changes.
- [x] Any dependent changes have been merged and published before.

## Git Log Checklist:

My commits logs have:

- [x] Subject starts with "subsystem|file: description".
- [x] Do not end the subject line with a period.
- [x] Limit the subject line to 50 characters.
- [x] Separate subject from body with a blank line.
- [x] Use the imperative mood in the subject line.
- [x] Wrap the body at 72 characters.
- [x] Use the body to explain what and why instead of how.

### CODE REVIEW

**Comment by AlonZivony** [comment]
This PR will require that from now on, all access to the output directory will be through the new utils functions.

**Comment by AlonZivony** [comment]
@rafaeldtinoco can you rerun the tests?
The error here is not logical, it might be some bug with the tests.

File: pkg/ebpf/events_processor.go [hunk 1]
```diff
@@ -446,3 +446,29 @@ func CopyFileByPath(src, dst string) error {
 	}
 	return nil
 }
+
+// CopyFileByRelativePath copies a file from src to dst, where destination is relative to a given directory
+func CopyFileByRelativePath(srcName string, dstDir *os.File, dstName string) error {
```
Comment (rafaeldtinoco) [inline pkg/ebpf/events_processor.go:451]: Why not to place in utils together with other functions you created ? Something different here ?

**Review by rafaeldtinoco — COMMENTED** [review]
Could not reproduce the test error here as well (tried go1.19 and go1.17). Error at: https://github.com/aquasecurity/tracee/runs/7987278937?check_suite_focus=true#step:4:406

**Review by rafaeldtinoco — COMMENTED** [review]
It all looks good to me. Left a comment about a helper function placement and I believe golang used in github actions is confused about file and package scopes because of some change.

**Review by rafaeldtinoco — APPROVED** [review]
LGTM (as it did before) but now passing tests (import changes to fix broken tests).

**Pull request merged**